Tests/Functional/Integration/ExtensionCreatorServiceTest.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace Integration;
+
+use PHPUnit\Framework\Attributes\DataProvider;
+use StefanFroemken\ExtKickstarter\Information\ExtensionInformation;
+use StefanFroemken\ExtKickstarter\Service\Creator\ExtensionCreatorService;
+use TYPO3\TestingFramework\Core\Functional\FunctionalTestCase;
+
+class ExtensionCreatorServiceTest extends FunctionalTestCase
+{
+    protected array $testExtensionsToLoad = [
+        'ext_kickstarter',
+    ];
+
+    protected array $coreExtensionsToLoad = [
+        'install',
+    ];
+
+    private string $tempDir;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->tempDir = sys_get_temp_dir() . '/extkickstarter_service_' . uniqid();
+        mkdir($this->tempDir, 0777, true);
+    }
+
+    protected function tearDown(): void
+    {
+        $this->deleteDirectory($this->tempDir);
+        parent::tearDown();
+    }
+
+    #[DataProvider('extensionCreationProvider')]
+    public function testItCreatesExpectedExtensionFiles(
+        ExtensionInformation $extensionInfo,
+        string $generatedPath,
+        string $expectedDir,
+        array $expectedFiles
```

Comment:
I have no problem with that, but since PHP 8.2 we have trailing commas after each argument, right? Is there php-cs-fixer directory missing for test directory?